.devcontainer/requirements.txt

```diff
@@ -18,3 +18,5 @@ ruff==0.4.6
 textalloc==1.1.3
```

Comment:
nice work to manage to collect this data
However  I find that roads and rivers alone are the perfect balance between too few and too much data, to still recognize the minimalist city's signature :thinking: 
I asked some friends who tend to agree with the fact that Railways aren't that recognizable and kind of pollute the whole minimalist design

Would you mind keeping your optim for the merge algorithm and the river improvment but discard the railway part ? :confused: 

It can be